app/src/androidTest/java/net/skyscanner/backpack/compose/calendar/BpkCalendarTest.kt

```diff
@@ -118,73 +88,14 @@ class BpkCalendarTest : BpkSnapshotTest() {
   }
 
   @Test
-  fun withStartDateSelected() {
```

Comment:
stopped relying on snapshot tests for any of the interactions. we were testing these twice - once setting things programmatically and once via interaction. left the programatical ones here